tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -33,97 +33,106 @@ public function testFix($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     * @param null|string $expectedWithKeepMultipleSpacesAfterComma
+     *
+     * @dataProvider testFixProvider
+     */
+    public function testFixKeepMultipleSpacesAfterComma($expected, $input = null, $expectedWithKeepMultipleSpacesAfterComma = null)
```

Comment:
It was my intention to keep everything in the same place (`testFixProvider`) in order to ease the development and maintance the code.
If you prefer I can split everything like [./Basic/BracesFixerTest.php](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1a801c0017c5b0544d00272179562f82586029b9/tests/Fixer/Basic/BracesFixerTest.php)